app/src/main/java/org/stepik/android/view/video_player/ui/activity/VideoPlayerActivity.kt

```diff
@@ -137,6 +139,8 @@ class VideoPlayerActivity : AppCompatActivity(), VideoPlayerView, VideoQualityDi
         playerView.setFastForwardIncrementMs(JUMP_TIME_MILLIS)
         playerView.setRewindIncrementMs(JUMP_TIME_MILLIS)
 
+        playerView.exo_controller.exo_fullscreen_icon.setOnClickListener { changeVideoRotation() }
```

Comment:
It can be used as `exo_fullscreen_icon` without `playerView.exo_controller.exo_fullscreen_icon` so caching will work